File: week6/pset6/mario.py
from cs50 import get_int

# check for valid height input
while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break
    
# print appropriate pyramid
for i in range(n):
    print(" " * (n - 1 - i), end='')
    print("#" * (i + 1), end='')
    print(" " * 2, end='')
    print("#" * (i + 1))

File: week1/pset1/mario.c
/*This is a programme that allows the user to select the height of a pyramid in a Mario obstacle course*/ 

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    
    do
    {
        n = get_int("Please select a height (1-8).\n");
    }
    while (n < 1 || n > 8);

    for (i = 0; i < n; i++)
    {
        printf(' ', n - 1 - i);
        printf('#', i + 1);
        printf(' ', 2);
        printf('#', i + 1);
    }
    
    printf('\n');
    
